extensions/arc/src/ui/dashboards/postgres/postgresParametersPage.ts

```diff
@@ -166,20 +155,19 @@ export class PostgresParametersPage extends DashboardPage {
 							cancellable: false
 						},
 						async (_progress, _token): Promise<void> => {
-							//Edit multiple
-							// azdata arc postgres server edit -n <server group name> -e '<parameter name>=<parameter value>, <parameter name>=<parameter value>,...'
 							try {
-								this.engineSettingUpdates!.forEach((value: string) => {
-									this.engineSettings += value + ', ';
+								this.parameterUpdates!.forEach((value, key) => {
+									engineSettings.push(`${key}=${value}`);
```

Comment:
I thought --engine-settings accepted a comma separated list. Are the settings comma separated somewhere?

Reply:
The array uses the toString method when passing into the command to show as comma separated.